plugins/modules/packaging/os/xbps.py

```diff
@@ -272,12 +277,11 @@ def main():
 
     module = AnsibleModule(
         argument_spec=dict(
-            name=dict(default=None, aliases=['pkg', 'package'], type='list'),
+            name=dict(default=None, aliases=['pkg', 'package'], type='list', elements='str'),
             state=dict(default='present', choices=['present', 'installed',
                                                    'latest', 'absent',
                                                    'removed']),
             recurse=dict(default=False, type='bool'),
-            force=dict(default=False, type='bool'),
```

Comment:
Simply removing it is not a good idea, this might break people's playbooks/roles. You can deprecate it (i.e. add `removed_in='3.0.0', removed_from_collection='community.general'` to the dict). This change would require a changelog fragment.